spot/reentrancy-demo.ipynb

```diff
@@ -0,0 +1,830 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "TODO: switch to a simpler demo model.\n",
+    "\n",
+    "# reentrancy-demo\n",
+    "\n",
+    "![](https://i.imgur.com/NYowq6j.png)\n",
+    "\n",
+    "<div style=\"background-color: #fff; box-shadow: 0 2px 4px 0 rgba(0,0,0,0.2); padding: 30px; padding-top: 24px; margin: 0px 40px\">\n",
+    "This Jupyter notebook is the code compliment to the blog posts <b><a href=\"https://spell.run/blog/automated-machine-failure-recovery-Xp3TEhEAACUAYwPM\">Automating GPU machine failure recovery in Google Compute Engine</a></b> and <b><a>Making cloud GPUs cost-effective using spot instances</a></b> (still forthcoming; working title).\n",
```

Comment:
are you intentionally committing the stuff in parenthesis?

Reply:
Yes, I will update this text when the next one comes out.